src/libstd/collections/hash/map.rs

```diff
@@ -522,11 +523,11 @@ impl<K, V, S> HashMap<K, V, S>
 
     // The caller should ensure that invariants by Robin Hood Hashing hold.
     fn insert_hashed_ordered(&mut self, hash: SafeHash, k: K, v: V) {
-        let raw_cap = self.raw_capacity();
+        let capacity = self.table.capacity();
```

Comment:
this is still the raw capacity (change is equivalent, so seems unnecessary to change).